modules/ROOT/pages/api-proxy-landing-page.adoc

```diff
@@ -33,6 +33,89 @@ API Proxy version 2.0.0 and later add native support for OAS API specification a
 
 You can specify the version of the proxy you want to use by clicking the *Advanced Options* drop-down menu and selecting the proxy version that best covers your needs.
 
+== API Proxy Validations
+
+You can configure RAML, REST, or WSDL proxies to perform validations on all incoming requests against the API definition you design. Depending on the Mule runtime and API Proxy versions you installed, you can modify these validation settings in API Manager and redeploy the proxy in Mule Runtime. 
+
+With Mule 4, you can apply strict validations as well as regular validations. for your API proxies. Additionally, you can also edit proxies directly from API Manager based on the API proxy version. For more information, see the <<default-setting-mule-4,Default Setting by API Proxy Versions>> table in this section.
+
+=== Regular Validations
+
+Regular validations enable applications to compare a request with API specifications.  As part of the regular validations for REST or RAML proxies, the payload (schema), query parameters, URI parameters, headers, and form parameters are validated. For WSDL proxies, the envelope is validated against the structure provided in the WSDL file. If the proxy receives a parameter that is not specified in the definition, that parameter is irrespectively sent to the backend service.
+
+
+=== Strict Validations
+
+Strict validations enable applications to compare a request with proxy specifications and allow only those query parameters that are specified in the API definition. Currently, you can perform strict validations for only query parameters. This feature is available only for RAML and WSDL proxies in Mule 4. 
```

Comment:
We could add that the feature by default is disabled and can be enalbed only for RAML and WSDL proxies in Mule 4.

Reply:
Done! Thanks!